hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/QueryHederaNativeOperations.java

```diff
@@ -153,4 +154,15 @@ public void trackSelfDestructBeneficiary(
             final AccountID deletedId, final AccountID beneficiaryId, @NonNull final MessageFrame frame) {
         throw new UnsupportedOperationException("Cannot track deletion in query context");
     }
+
+    /**
+     * Refuses to get the authorizing simple keys.
```

Comment:
wrong comments?

Reply:
I guess copilot had different opinion on the code bellow 😀